bin/helpers/capabilityHelper.js

```diff
@@ -91,6 +93,8 @@ const validate = (bsConfig) => {
 
     if(!bsConfig.run_settings.cypress_proj_dir) reject(Constants.validationMessages.EMPTY_SPEC_FILES);
 
+    if (!fs.existsSync(path.join(process.cwd(), bsConfig.run_settings.cypress_proj_dir, 'cypress.json'))) reject(Constants.validationMessages.EMPTY_CYPRESS_JSON);
```

Comment:
Can we also include the path as well in the error message `bsConfig.run_settings.cypress_proj_dir`?

Reply:
Have also added the Cypress Project Directory in the error message.